frontend/packages/ceph-storage-plugin/integration-tests-cypress/views/bc.ts

```diff
@@ -97,28 +157,71 @@ const setBackingStores = (tiers: Tier[]) => {
       expect($items).toHaveLength(2);
     });
   }
-  selectBackingStore(1, tests.pop());
+  selectStoreFromTable(1, tests.pop());
   if (TierCountMap[tiers[0]] > 1) {
-    selectBackingStore(1, tests.pop());
+    selectStoreFromTable(1, tests.pop());
   }
   // Select tier 2 Backing Stores
   if (tiers.length > 1) {
-    selectBackingStore(2, tests.pop());
+    selectStoreFromTable(2, tests.pop());
     if (TierCountMap[tiers[1]] > 1) {
-      selectBackingStore(2, tests.pop());
+      selectStoreFromTable(2, tests.pop());
     }
   }
 };
 
-export const createBC = (tiers: Tier[]) => {
-  setGeneralData();
-  cy.contains('Next').click();
-  // Placement Policy Page
-  setPlacementPolicy(tiers);
+const selectItemFromStoreDropdown = (name: string, type: StoreType) => {
+  cy.byTestID(`${type === StoreType.NamespaceStore ? 'nns' : 'nbs'}-dropdown-toggle`).click();
+  cy.byTestID(`${name}-dropdown-item`).click();
+};
+
+const NamespaceBucketClassFlows = {
```

Comment:
Can we name it better, something related to the action ?

Reply:
use switch() since all over we are using functions. It will be consistent therefore.